### ISSUE TITLE
Training data format

### ISSUE THREAD

**Issue by hoangtnm** [junleen/RainNet#16]
Hi, when training RainNet, does it need any grounth truth labels (real images), or just 2 pair of composite image and its segmentation mask is enough? thank you

**Comment by junleen** [comment]
When you are training the network, you should provide the ground truth images as the targets. We didn't use two pairs of composite images and their segmentation masks, just 1 pair.
